workflows/argo-events/workflowtemplates/get-bmc-creds.yaml

```diff
@@ -19,43 +18,49 @@ spec:
       inputs:
         parameters:
           - name: device_id
+      outputs:
+        parameters:
+          - name: secret
+            valueFrom:
+              expression: "tasks['get-bmc-creds'].outputs.parameters.secret"
+      dag:
+        tasks:
+          - name: get-bmc-ip
+            templateRef:
+              name: get-bmc-ip
+              template: main
+            arguments:
+              parameters:
+                - name: device_id
+                  value: "{{inputs.parameters.device_id}}"
+          - name: get-bmc-creds
+            template: get-standard-bmc-creds
+            arguments:
+              parameters:
+                - name: ip_address
+                  value: "{{tasks.get-bmc-ip.outputs.parameters.ip}}"
+    - name: get-standard-bmc-creds
+      inputs:
+        parameters:
+          - name: ip_address
       outputs:
         parameters:
           - name: secret
             valueFrom:
               path: "/tmp/output.txt"
-      script:
-        image: ghcr.io/rackerlabs/understack/argo-utils-python3.11.8:latest
+      container:
+        image: ghcr.io/rss-engineering/undercloud-deploy/argo-undercloud-rackspace3.11.8:latest
```

Comment:
This should be a container from this repo.

Reply:
fixed in 5c8d61c